style.css

```diff
@@ -0,0 +1,95 @@
+body {
+	font-family: sans-serif;
+	margin: 0px;
+}
+
+nav {
+	margin: 20px;
+	text-align: left;
+	font-size: 20px;
+	padding-right: 10px;
+	padding-left: 10px;
+}
+
+a {
+	color: black
+}
+
+a:hover {
+	color: grey;
+
+}
+
+h1 {
+	color: white;
+	font-size: 60px;
+	position: absolute;
+	top: 210px;
+	padding-left: 60px;
+	line-height: 130%;
+}
+
+h3 {
+	color: white;
+	font-size: 70px;
+	position: absolute;
+	top: 750px;
+	margin: 40px;
+	background-position: right;
+}
+
+.gollum {
+	position: relative;
+	width: 100%;
+}
+
+.gollumtwo {
+	position: relative;
+	width: 100%;
+}
+
+.list {
+	margin: 30px;
+	color: white;
+	text-align: left;
+	position: absolute;
+	top: 1600px;
+	line-height: 130%;
+}
+
+h3.list {
+	margin: 20px;
+	color: white;
+	font-size: 70px;
+	text-align: left;
+	position: absolute;
+	top: 1500px;	
+}
+
+.rings {
+    position: relative;
+    float: left;
+    width:  300px;
+    height: 300px;
```

Comment:
background-size, and background-position are not needed in this situation. It is generally used when you load images with css.